### ISSUE TITLE
Lua app fonts being overwritten on tracks that have font enabled by lua

### ISSUE THREAD

**Issue by sollrakc** [ac-custom-shaders-patch/acc-extension-config#370]
I found this issue in the GT7 HUD app while using a track that has fonts enabled by lua, like Gunnar333's Sachsenring.

Apparently, it's something to do with a draw order issue for imgui, according to chése.

**Comment by Marocco2** [comment]
What Lua apps do you use?

**Comment by sollrakc** [comment]
No, the same thing happens with cars that use lua scripts for the clock display as well. I've tested with Seb/Amy's Honda Integra from SRP and it's the same issue. Both the car and Sachsenring are using loading a font in .ttf format via lua scripts. Seems like a font leaking problem.

**Comment by C1XTZ** [comment]
Apparently apps are affected by this as well
The "Mobilephone" app gets loaded after the other and fails to load its font, this is on 1.80pre150 (2295)

![image](https://user-images.githubusercontent.com/107640769/229306948-9ca1fd6a-08f0-49bc-a764-449d5388ace1.png)

```lua
--example of both of the apps use fonts
local appfont =  ui.DWriteFont('IBM Plex Sans', '.')
ui.pushDWriteFont(appfont)
ui.dwriteText('test', 60, rgbm.color.white)
ui.popDWriteFont()
```

**Comment by C1XTZ** [comment]
Should be fixed with v0.1.80-preview346 (my apps are), can you test and confirm when you get the chance? @sollrakc
